packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -175,7 +177,19 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
         logger.debug(`Cache: GET ${key}`, entry)
         const reqMaxAge = _getRequestMaxAge(data)
         if (reqMaxAge && reqMaxAge !== entry.maxAge) await _cacheOnSuccess(entry)
-        return { jobRunID: data.id, ...entry }
+        const ttl = await cache.pttl(key)
```

Comment:
This looks like a typo here. It should be `cache.ttl`.

Reply:
To get the ttl is milliseconds, `redis` uses the `pttl` [command](https://redis.io/commands/PTTL).

Reply:
Ok, not sure why they have two commands to get the same info in different resolutions heh

Maybe we call it `ttl`, but internally in Redis cache call `pttl` and for in memory cache we make sure we return the `ttl` in ms.